canonical-web-and-design/maas-ui
# fix(legacy): fix controllers when editing VLANs

## Done

- Show the correct controllers when editing a VLAN.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the controller list and click on a controller.
- Click on the VLANs tab.
- Click on a VLAN in the VLAN column.
- In the Summary section note the controllers listed next to `Rack controller(s)`.
- Click the Edit button.
- the rack controllers should be the same as before.

## Fixes

Fixes: #2572.
